CalcualtorLogic.js

```diff
@@ -28,6 +28,8 @@ class Calculator {
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement("div")
```

Comment:
The `wrapperDiv` needs the same CSS class on the previous HTML
![image](https://user-images.githubusercontent.com/16171515/108383582-d04a5c00-7201-11eb-8cb1-86acea3a1189.png)

You can use the `classList` API to add the required class name to The `wrapperDiv` 
![image](https://user-images.githubusercontent.com/16171515/108383938-28815e00-7202-11eb-9e3f-3c351f4f7afc.png)

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList